### PR TITLE
Data sample

### PR DESCRIPTION
**Pull request by joshuaellinger**
Resolves Issue #13.

This is output of the Google PUSH logic.  This sample file should be used for writing unit a test of the post logic.

The code lives in the data-entry repo.

We should open up a different Issue to actually wiring it to the API endpoint once the unit test against the sample is successful.

### CODE REVIEW

File: tests/app/sample_data.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (julia326) [inline tests/app/sample_data.json:1]: what's the difference between `sample_data` and `sample_batch`? Do you mean batch in the table name sense?

Comment (joshuaellinger) [reply tests/app/sample_data.json:1]: this is an obsolete older version.  I'll remove it.  good catch

**Comment by joshuaellinger** [comment]
Any other issues?

**Comment by joshuaellinger** [comment]
Closing per Julia's feedback in slack.

**Pull request closed without merging**